Buildings/ThermalZones/Detailed/Validation/BESTEST/Data/CriteriaLimits.mo

```diff
@@ -0,0 +1,21 @@
+within Buildings.ThermalZones.Detailed.Validation.BESTEST.Data;
+record CriteriaLimits
+  "Record that is used for summary of the test acceptance criteria limits"
+  extends Modelica.Icons.Record;
+  parameter Modelica.Units.SI.Energy LowerLimit "Lower limit of the test acceptance criteria";
+  parameter Modelica.Units.SI.Energy UpperLimit "Upper limit of the test acceptance criteria";
```

Comment:
Better use `lowerLimit` and `upperLimit` to follow naming convention.

Reply:
Updated.